enterprise/views.py

```diff
@@ -1265,21 +1201,37 @@ def get_enterprise_program_enrollment_page(self, request, enterprise_customer, p
         elif discount_data.get('total_incl_tax_excl_discounts') is None:
             messages.add_missing_price_information_message(request, program_title)
 
-        # Update our context with the above calculated details and more.
-        context_data = self.context_data.copy()
```

Comment:
did you intend to remove this line?

Reply:
yes, this is intentional, translations (in order to work) needed to be moved into the method instead of being defined as the class attribute.